### ISSUE TITLE
Tests for twisted.news should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7703]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7703|
|Type|enhancement|
|Created|2014-11-02 11:25:38Z|
|Branch|https://github.com/twisted/twisted/tree/news-tests-names-7703|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7703.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/238/238734c2110abb2817eb7ec469f440e14795e7a8/40c5b153813bb8391ea59e2aef358e53379dffda.patch) (1005 bytes) - added by eeshangarg on 2014-11-02 11:34:46Z - 
* [bugfix_7703_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/238/238734c2110abb2817eb7ec469f440e14795e7a8/118d104e7bbab39dd0922e73b4e3d9c011b18443.patch) (1112 bytes) - added by eeshangarg on 2014-11-17 16:37:44Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7703 7703
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_news_tests_names_7703 branches/news-tests-names-7703
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__news news
keywords__easy easy
time__1414927538606051 1414927538606051
changetime__1417022849340537 1417022849340537
version__None None
owner__exarkun exarkun
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg set owner to @glyph|
|-|-|

Hi! I just added a patch that changes all of the tests in twisted.news to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks!
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph removed owner|
|-|-|

Un-assigning all review tickets assigned to me.  Unless there's a specific reason _I_ need to review something, let's leave them unassigned :-).

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

It looks good, but it needs  the newsfile in order to have it merged. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_4"></a>|@eeshangarg commented|
|-|-|

Hello!

I just added a new patch called bugfix_7703_2.patch that includes the .misc news file. Thanks a lot for the review. :)

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [43503]) Branching to 'news-tests-names-7703'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [43504]) Apply bugfix_7703_2.patch 

Refs [#7703](https://github.com/twisted/twisted/issues/7703)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Thanks.  These changes look fine to me as well.  I'll merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set status to `closed`|
|-|-|

(In [43505]) Merge news-tests-names-7703

Author: eeshangarg
Reviewer: adiroiban, exarkun
Fixes: [#7703](https://github.com/twisted/twisted/issues/7703)

Rename all of the test case classes in Twisted News to conform to the
recently documented test case class naming convention.
